### ISSUE TITLE
`StringContainer.add_from_etree` fails for empty node

### ISSUE THREAD

**Issue by notEvil** [pretix/python-drafthorse#53]
Hi,

a perfectly valid invoice may contain for instance `<IncludedNote><Content/></IncludedNote>`. `Container.add` fails in this case as `root.text` is `None`. The following solves this issue:

```diff
--- models/container.py.orig	2024-12-11 10:20:20.833947854 +0100
+++ models/container.py	2024-12-11 10:20:24.930598120 +0100
@@ -91,4 +91,5 @@
         el._text = child
 
     def add_from_etree(self, root):
-        self.add(root.text)
+        if (_text := root.text) is not None:
+            self.add(_text)
```

**Comment by raphaelm** [comment]
I don't think that's a perfectly valid invoice then, at least not according to EN16931 or XRechnung standards:

https://test-docs.peppol.eu/pint/pint-eu/pint-eu/trn-creditnote/rule/PEPPOL-EN16931-R008/

**Comment by notEvil** [comment]
You are right. The reason I thought its valid is that Xsd and Schematron validation using official files from ZUGFeRD 2.2 and 2.3.2 succeeds.
